README.md

```
@@ -1,112 +1,34 @@
 Using the Trustless Work API, this project builds an escrow-based solution to streamline crop financing—an innovative step for decentralized agriculture. What’s even more exciting? We integrated HyperCycle NFC rings to release funds from escrow, marking this as our first-ever DePIN (Decentralized Physical Infrastructure) integration!
 
-This project was created and initiated during the [Ethereum Pura Vida 2024](https://devfolio.co/projects/revolutionary-farmers-b88c) hackathon.
+This project was created and initiated during the Ethereum Pura Vida 2024 hackathon.
 
-## 🗂️ Project Structure
+## 📖 Full Documentation
+For a more detailed and structured guide on the project, including setup instructions, development guidelines, and more, please visit our official GitBook:
 
-- **android**: An Android Studio app where users can approve trust transactions using the Zero-Knowledge Ring ([Zero-Knowledge Ring](https://www.zero-knowledge-ring.com/)) for privacy-focused transaction validation. This app also connects to the Trustless Work API to manage escrow-based approvals.
-- **contracts**: Blockchain smart contracts (under development) that will support decentralized management of the trusts, providing a transparent and secure solution for crop financing.
+➡️ Revolutionary Farmers GitBook: https://revolutionary-farmers.gitbook.io/revolutionary-farmers
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)